beer_data/build.gradle.kts

```diff
@@ -0,0 +1,18 @@
+plugins {
+    id("billionbeers.android.library")
+    id("billionbeers.android.hilt")
+}
+
+android {
+    namespace = "com.simtop.beer_data"
+}
+
+dependencies {
+    implementation(project(":core"))
+    implementation(project(":beerdomain"))
+    implementation(project(":beer_database"))
+    implementation(project(":beer_network"))
+    
+    implementation(libs.javax.inject)
```

Comment:
Or this is not needed or the hilt part is not needed, check the DI Module of this, I use inject when we don't have DI modules but we need inject stuff.